### PR TITLE
File Manager - Add sorting items by filetype in details view

### PR DESCRIPTION
**Pull request by Staziz**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.dataGrid/columnsHeadersView.tests.js [hunk 1]
```diff
@@ -792,18 +792,18 @@ QUnit.test('Apply sorting by click', function(assert) {
     // arrange
     var testElement = $('#container');
 
-    $.extend(this.columns, [{ index: 0, alignment: 'right', allowSorting: true }]);
+    $.extend(this.columns, [{ index: 0, alignment: 'right', allowSorting: true }, { index: 1, dataField: "isDirectory", caption: 'Column 2', allowSorting: true }]);
```
Comment (maxerokh) [inline testing/tests/DevExpress.ui.widgets.dataGrid/columnsHeadersView.tests.js]: You have written tests in data grid fixture. The tests test data grid functionality, but we should test file manager features.

Please, create new test that create file manager and test sorting of data grid among file manager enviroment.

**Review by maxerokh — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.ui.widgets.dataGrid/columnsHeadersView.tests.js (hunk 1 above)
Comment (Staziz) [reply testing/tests/DevExpress.ui.widgets.dataGrid/columnsHeadersView.tests.js]: Tests are moved to file_manager/detailsView.tests 
Check it, please.

File: testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js [hunk 2]
```diff
@@ -143,4 +143,53 @@ QUnit.module("Details View", moduleConfig, () => {
...
+
+        assert.equal(columnHeader.find(".dx-sort-up").length, 0);
+        assert.equal(columnHeader.find(".dx-sort-down").length, 0, "sort elements are absent");
+
+        assert.equal(columnHeader.attr("aria-sort"), "none", "sorting default");
+
+        columnHeader.trigger("dxclick");
+        this.clock.tick(400);
+
+        assert.ok(getCellValueInDetailsView(this.$element, 1, 2).indexOf("1.txt") === 0);
```
Comment (maxerokh) [inline testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js]: Use the assert.equal method, to check strings

File: testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js [hunk 3]
```diff
@@ -143,4 +143,53 @@ QUnit.module("Details View", moduleConfig, () => {
         assert.equal(eventCounter, 1);
     });
 
+    test("Apply sorting by click", function(assert) {
```
Comment (maxerokh) [inline testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js]: You test that column with thumb was sorted. I think you should rename the test.

File: testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js (hunk 2 above)
Comment (maxerokh) [inline testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js]: Use the assert.equal method in these cases. This method allow more effectively find errors.

File: testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js [hunk 4]
```diff
@@ -143,4 +143,53 @@ QUnit.module("Details View", moduleConfig, () => {
...
+
+        var fileManagerDetails = $(document).find(".dx-filemanager-details");
```
Comment (maxerokh) [inline testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js]: Use `const` insead of `var`

File: testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js [hunk 5]
```diff
@@ -143,4 +143,53 @@ QUnit.module("Details View", moduleConfig, () => {
...
+        assert.equal(fileManagerDetails.length, 1, "details view");
+
+        var headersRow = fileManagerDetails.find(".dx-header-row");
+        assert.ok(headersRow.is("tr"), "headers row");
+
+        var columnHeader = headersRow.find("td").filter(function() {
```
Comment (maxerokh) [inline testing/tests/DevExpress.ui.widgets/fileManagerParts/detailsView.tests.js]: May be use constant instead of regular expression..

**Review by maxerokh — CHANGES_REQUESTED** [review]

**Review by maxerokh — APPROVED** [review]

**Pull request merged**